tests/test_engine.py

```diff
@@ -21,10 +21,10 @@
 
 test_data_folder = join(dirname(abspath(__file__)), 'data')
 test_pallets_folder = join(test_data_folder, 'list_pallets')
-config.config_location = config_location = join(abspath(dirname(__file__)), 'config.json')
+config_location = config_location = join(abspath(dirname(__file__)), 'config.json')
 
 
-class TestConfigInit(unittest.TestCase):
+class CleanUpAlternativeConfig(unittest.TestCase):
```

Comment:
this seems like a good spot for a pytest.fixture no? Your implementation is fine but I am curious why you chose to take two different approaches for seemingly the same issue.

Reply:
Ya, it probably is. The reason why they are different is because I wasn't familiar with pytest fixtures until after I wrote this code.

Reply:
ok maybe for another time